<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HOME" ignored="1" name="Default test(failed)(failed)" tests="2" failures="2" timestamp="09 May 2019 18:07:36 GMT" time="0.089" errors="0">
  <testcase name="failureMethod" time="0.015" classname="com.test.SeleniumPrograms.ExecuteFailedTestCases">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.test.SeleniumPrograms.ExecuteFailedTestCases.failureMethod(ExecuteFailedTestCases.java:16)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- failureMethod -->
  <testcase name="failureMethod2" time="0.0" classname="com.test.SeleniumPrograms.ExecuteFailedTestCases">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.test.SeleniumPrograms.ExecuteFailedTestCases.failureMethod2(ExecuteFailedTestCases.java:28)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- failureMethod2 -->
  <testcase name="passMethod" time="0.0" classname="com.test.SeleniumPrograms.ExecuteFailedTestCases">
    <ignored/>
  </testcase> <!-- passMethod -->
</testsuite> <!-- Default test(failed)(failed) -->
